test/micro/org/openjdk/bench/vm/compiler/InstructionScheduling.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022, Loongson Technology. All rights reserved.
```

Comment:
Hi @sunny868 ,

We already have the copyright header for Loongson like the following.
```
./hotspot/jtreg/compiler/print/PrintCompileQueue.java: * Copyright (c) 2019, Loongson Technology Co. Ltd. All rights reserved.
./hotspot/jtreg/compiler/compilercontrol/CompilationModeHighOnlyTest.java: * Copyright (c) 2019, Loongson Technology Co. Ltd. All rights reserved.
./hotspot/jtreg/compiler/profiling/TestProfileCounterOverflow.java: * Copyright (c) 2019, Loongson Technology Co. Ltd. All rights reserved.
```

I'm not sure whether it had been changed.

Reply:
> Hi @sunny868 ,
> 
> We already have the copyright header for Loongson like the following.
> 
> ```
> ./hotspot/jtreg/compiler/print/PrintCompileQueue.java: * Copyright (c) 2019, Loongson Technology Co. Ltd. All rights reserved.
> ./hotspot/jtreg/compiler/compilercontrol/CompilationModeHighOnlyTest.java: * Copyright (c) 2019, Loongson Technology Co. Ltd. All rights reserved.
> ./hotspot/jtreg/compiler/profiling/TestProfileCounterOverflow.java: * Copyright (c) 2019, Loongson Technology Co. Ltd. All rights reserved.
> ```
> 
> I'm not sure whether it had been changed.

typo